app/views/validation_requests/index.html.erb

```diff
@@ -133,23 +67,23 @@
         <p class="govuk-body">
           Send further requests for this application. Any new requests will be immediately sent.
         </p>
-      <% elsif @planning_application.can_invalidate? %>
+      <% elsif @planning_application.not_started? %>
```

Comment:
are we sure this is the one we want here? I remember the can_invalidate being necessary for some reason.. will UAT in the front end

Reply:
looks great actually :D had a full on uat, as far as I can see its good to go!